### PR TITLE
SAMZA-1236: Initial draft of the fluent API example for tutorials

### PR DESCRIPTION
**Pull request by jmakes**

### CODE REVIEW

**Comment by jmakes** [comment]
@prateekm @vjagadish1989
It's ready for review

File: src/main/config/wikipedia-application.properties [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# Kafka System
+systems.kafka.samza.factory=org.apache.samza.system.kafka.KafkaSystemFactory
+systems.kafka.consumer.zookeeper.connect=localhost:2181/
+systems.kafka.producer.bootstrap.servers=localhost:9092
+systems.kafka.default.stream.replication.factor=1
+systems.kafka.default.stream.samza.msg.serde=json
+
+# Streams which are not on default system or have special characters in the physical name.
+streams.en-wikipedia.samza.system=wikipedia
+streams.en-wikipedia.samza.physical.name=#en.wikipedia
```
Comment (prateekm) [inline src/main/config/wikipedia-application.properties]: Is the name intentionally empty?

Comment (jmakes) [inline src/main/config/wikipedia-application.properties]: That's really weird. The job is able to parse the whole name, including "#"
Works with task.inputs in the old hello-samza as well

**Comment by jmakes** [comment]
Prototype merge script didn't close this PR. Closing manually

**Pull request closed without merging**